### PR TITLE
Parser URL added

### PR DESCRIPTION
**Pull request by rajendrakhabiyagate6**
Fixes #
- Parser function short url is not working so updated it with git url

### CODE REVIEW

File: Solutions/Lookout/Workbooks/LookoutEvents.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     {
       "type": 1,
       "content": {
-        "json": ">**NOTE:** This workbook uses a parser based on a Kusto Function to normalize fields. [Follow these steps](https://aka.ms/sentinel-lookoutapi-parser) to create the Kusto function alias **LookoutEvents**."
+        "json": ">**NOTE:** This workbook uses a parser based on a Kusto Function to normalize fields. [Follow these steps](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Lookout/Parsers/LookoutEvents.txt) to create the Kusto function alias **LookoutEvents**."
```
Comment (v-maudan) [inline Solutions/Lookout/Workbooks/LookoutEvents.json:7]: I have created short link for parser. https://aka.ms/sentinel-lookoutapi-parser. Please revert this changes. Thanks!

File: Solutions/Lookout/Data Connectors/Lookout_API_FunctionApp.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
 			"description": ">**NOTE:** This connector uses Azure Functions to connect to the Lookout REST API to pull its events into Azure Sentinel. This might result in additional data ingestion costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
 		},
 		{
-			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-lookoutapi-parser) to create the Kusto functions alias, **LookoutEvents**"
+			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Lookout/Parsers/LookoutEvents.txt) to create the Kusto functions alias, **LookoutEvents**"
```
Comment (v-maudan) [inline Solutions/Lookout/Data Connectors/Lookout_API_FunctionApp.json:71]: @rajendrakhabiyagate6  I have created short link for parser. https://aka.ms/sentinel-lookoutapi-parser
Please update description field value as follows.
` >**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**LookoutEvents**](https://aka.ms/sentinel-lookoutapi-parser) which is deployed with the Azure Sentinel Solution.`

File: Solutions/Lookout/Data Connectors/Lookout_API_FunctionApp.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
 	"title": "Lookout",
 	"publisher": "Lookout",
 	"descriptionMarkdown": "The [Lookout](https://lookout.com) data connector provides the capability to ingest [Lookout](https://mtp.lookout.com/) events into Azure Sentinel through the REST API. Refer to [API documentation](https://mtp.lookout.com) for more information. The connector provides ability to get events which helps to examine potential security risks and more.",
-	"additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. ​Follow the steps to use this Kusto functions alias **LookoutEvents** in queries and workbooks [Follow steps to get this Kusto functions>](https://aka.ms/sentinel-lookoutapi-parser).",
+	"additionalRequirementBanner": "These queries and workbooks are dependent on a parser based on Kusto to work as expected. ​Follow the steps to use this Kusto functions alias **LookoutEvents** in queries and workbooks [Follow steps to get this Kusto functions>](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Lookout/Parsers/LookoutEvents.txt).",
```
Comment (v-maudan) [inline Solutions/Lookout/Data Connectors/Lookout_API_FunctionApp.json:6]: @rajendrakhabiyagate6  I have created short link for parser. https://aka.ms/sentinel-lookoutapi-parser
Please update additionalRequirementBanner filed value as follows.
` This data connector depends on a parser based on a Kusto Function to work as expected [**LookoutEvents**](https://aka.ms/sentinel-lookoutapi-parser) which is deployed with the Azure Sentinel Solution.`

**Review by v-maudan — CHANGES_REQUESTED** [review]

**Comment by rajendrakhabiyagate6** [comment]
No More needed, Created Short url is same as we used
